package.json

```diff
@@ -94,7 +94,7 @@
     "node": "^20.17.0 || >=22.9.0",
     "pnpm": "10.x"
   },
-  "packageManager": "pnpm@10.10.0",
+  "packageManager": "pnpm@10.11.0+sha512.6540583f41cc5f628eb3d9773ecee802f4f9ef9923cc45b69890fb47991d4b092964694ec3a4f738a420c918a333062c8b925d312f42e4f0c263eb603551f977",
```

Comment:
this would have to be rolled back, I would prefer to keep pnpm minimum version to v10.10